build/Microsoft.DotNet.Cli.Prepare.targets

```diff
@@ -42,6 +43,17 @@
     <PropertyGroup>
       <SdkVersion>@(BuildVersion ->'%(NugetVersion)')</SdkVersion>
       <SdkBrandName>Microsoft .NET Core 1.0.0 - SDK Preview 2</SdkBrandName>
+
+      <ArtifactNameSdk>dotnet-sdk</ArtifactNameSdk>
```

Comment:
The quick and dirty way I did this, I feel is an anti-pattern, but I don't know how to do it the "MSBuild way".  It was a lot of duplication.  I have a PR short-coming for "Init".